### PR TITLE
First draft of a organisation README.md

### LINKED ISSUE #3467: Create a README for the massalabs organisation
Create a custom page for http://github.com/massalabs/. It can improve user journey!

### PR DESCRIPTION
**Pull request by litchipi**
Closes: https://github.com/massalabs/massa/issues/3467

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -1 +1,46 @@
...
+- [thyra-plugin-blogNoCode](https://github.com/massalabs/thyra-plugin-blogNoCode): Plugin for Thyra to create a blog without coding, allowing to store the website on the blockchain.
+- [thyra-node-manager-plugin](https://github.com/massalabs/thyra-node-manager-plugin): Plugin for Thyra to manage a remote node from a user-friendly interface.
+- [thyra-playground-plugin](https://github.com/massalabs/thyra-playground-plugin): Plugin for Thyra to serve a playground for smart-contracts development.
+- [thyra-plugin-massa-core](https://github.com/massalabs/thyra-plugin-massa-core): Plugin for Thyra implementing Massa core features.
+- [thyra-plugin-hello-world](https://github.com/massalabs/thyra-plugin-hello-world): HelloWorld example for plugin developers.
+- [thyra-menu-bar-app](https://github.com/massalabs/Thyra-Menu-Bar-App): Menu bar application icon for Thyra.
+
+## Tests and benchmarks
+
+- [massa-benchmark](https://github.com/massalabs/massa-benchmark-main): Contains benchmarks of some parts of the massa blockchain.
```
Comment (AurelienFT) [inline profile/README.md:39]: This is a private repo and so shouldn't be mentioned here

**Review by AurelienFT — COMMENTED** [review]
For the core side the description are ok.

@massalabs/run and @SlnPons or @longgamma-massa can you check that the description and the repository listed covers everything you have to offer to the public (maybe some are not fully update but I don't think we should mention it there. I think it should be on the readme of each repository)

File: profile/README.md [hunk 2]
```diff
@@ -1 +1,46 @@
 # Massa
+
+Welcome aboard on the *Massa* organisation. On this profile you will find every repositories related to the development on the **Massa blockchain**.
```
Comment (longgamma-massa) [inline profile/README.md:3]:
```suggestion
Welcome aboard to *Massa* organisation. On this profile you will find all repositories related to the development on **Massa blockchain**.
```

File: profile/README.md [hunk 3]
```diff
@@ -1 +1,46 @@
...
+
+## Code
+
+- [massa](https://github.com/massalabs/massa): Main code of the Massa blockchain, builds the node and client, holds all the critical code for the architecture.
```
Comment (longgamma-massa) [inline profile/README.md:7]:
```suggestion
- [massa](https://github.com/massalabs/massa): Main code of Massa blockchain, builds the node and client, holds all the critical code for the architecture.
```

File: profile/README.md [hunk 4]
```diff
@@ -1 +1,46 @@
...
+- [massa-proto](https://github.com/massalabs/massa-proto): Repository centralizing protobuf bindings for Massa GRPC.
```
Comment (longgamma-massa) [inline profile/README.md:8]:
```suggestion
- [massa-proto](https://github.com/massalabs/massa-proto): A repository centralizing protobuf bindings for Massa GRPC.
```

File: profile/README.md [hunk 5]
```diff
@@ -1 +1,46 @@
...
+- [massa-bridge](https://github.com/massalabs/massa-bridge): Blockchain bridge between Massa and any EVM-compatible blockchains.
```
Comment (longgamma-massa) [inline profile/README.md:9]:
```suggestion
- [massa-bridge](https://github.com/massalabs/massa-bridge): Blockchain bridge between Massa and any EVM-compatible blockchain.
```

File: profile/README.md [hunk 6]
```diff
@@ -1 +1,46 @@
...
+- [PeerNet](https://github.com/massalabs/PeerNet): P2P network backend for the Massa blockchain.
```
Comment (longgamma-massa) [inline profile/README.md:10]:
```suggestion
- [PeerNet](https://github.com/massalabs/PeerNet): A P2P network backend for Massa blockchain.
```

File: profile/README.md [hunk 7]
```diff
@@ -1 +1,46 @@
...
+- [stream_limiter](https://github.com/massalabs/stream_limiter): Rust library to implement read/write speed limits on any stream.
+
+## Smart contracts
+
+- [massa-sc-runtime](https://github.com/massalabs/massa-sc-runtime): Sandbox to run the smart-contracts (compiled as WASM) on a Massa node.
+- [massa-as-sdk](https://github.com/massalabs/massa-as-sdk): SDK to build smart-contracts in AssemblyScript. Abstracts the Massa's ABI function.
+- [massa-sc-toolkit](https://github.com/massalabs/massa-sc-toolkit): Provide tools such as `@massalabs/sc-project-initializer` to improve the experience of smart-contracts development.
+- [massa-sc-compiler](https://github.com/massalabs/massa-sc-compiler): Compile a smart-contract to a WASM binary.
```
Comment (longgamma-massa) [inline profile/README.md:18]:
```suggestion
- [massa-sc-runtime](https://github.com/massalabs/massa-sc-runtime): Sandbox to run smart contracts (compiled as WASM), on a Massa node.
- [massa-as-sdk](https://github.com/massalabs/massa-as-sdk): An SDK to build smart contracts in AssemblyScript. Abstracts Massa's ABI functions.
- [massa-sc-toolkit](https://github.com/massalabs/massa-sc-toolkit): Provides tools such as `@massalabs/sc-project-initializer` to improve the smart-contract development experience.
- [massa-sc-compiler](https://github.com/massalabs/massa-sc-compiler): Compile a smart contract to WASM binary.
```

File: profile/README.md [hunk 8]
```diff
@@ -1 +1,46 @@
...
+- [massa-sc-playground](https://github.com/massalabs/massa-sc-playground): Playground in AssemblyScript for smart-contracts.
```
Comment (longgamma-massa) [inline profile/README.md:19]: @Ben-Rey @AurelienFT not sure that this repo needs to be mentioned. If there was no developments after October on this topic, I suggest we deprecate the repo and remove SC-playground from Documentation

File: profile/README.md [hunk 9]
```diff
@@ -1 +1,46 @@
...
+- [massa-web3](https://github.com/massalabs/massa-web3): Library to help the implementation of Web3 applications on top of the Massa blockchain.
```
Comment (longgamma-massa) [inline profile/README.md:20]:
```suggestion
- [massa-web3](https://github.com/massalabs/massa-web3): A library to help the implementation of Web3 applications on top of Massa blockchain.
```

File: profile/README.md [hunk 10]
```diff
@@ -1 +1,46 @@
...
+- [massa-as-gaming-sdk](https://github.com/massalabs/massa-as-gaming-sdk): SDK for smart-contracts in AssemblyScript made for game development.
+- [massa-standards](https://github.com/massalabs/massa-standards): Code implementing standard smart-contracts, such as NFT or ERC-20.
+- [massa-sc-examples](https://github.com/massalabs/massa-sc-examples): Smart contract examples.
+- [as](https://github.com/massalabs/as): Utils to write smart-contracts in AssemblyScript.
```
Comment (longgamma-massa) [inline profile/README.md:24]:
```suggestion
- [massa-as-gaming-sdk](https://github.com/massalabs/massa-as-gaming-sdk): An SDK for smart contracts in AssemblyScript, made for game development.
- [massa-standards](https://github.com/massalabs/massa-standards): Code implementing standard smart contracts, such as NFT or ERC-20.
- [massa-sc-examples](https://github.com/massalabs/massa-sc-examples): Smart-contract examples.
- [as](https://github.com/massalabs/as): Utils to write smart contracts in AssemblyScript.
```

Comment (longgamma-massa) [reply profile/README.md:24]: I fixed the language, however I doubt these are good descriptions.
Massa-standards -> is not only made for showing SC-standards like NFT and ERC-20, but they are intended to host all Massalabs standards (check the repo, there's wallet standard apart from SCs).

File: profile/README.md [hunk 11]
```diff
@@ -1 +1,46 @@
...
+## Tests and benchmarks
+
+- [massa-benchmark](https://github.com/massalabs/massa-benchmark-main): Contains benchmarks of some parts of the massa blockchain.
+- [massa-unit-tests-src](https://github.com/massalabs/massa-unit-tests-src): Source of every smart-contracts serving as unit tests inside the [massa-core](https://github.com/massalabs/massa) repository.
+- [massa-sc-tester](https://github.com/massalabs/massa-sc-tester): Minimal testing environment for smart-contracts.
+
+## Documentation
+
+- [massa-docs](https://github.com/massalabs/massa-docs): Contains the documentation and code to build [the massa-docs website](https://docs.massa.net/en/latest/).
+- [docu-dev](https://github.com/massalabs/docu-dev): Documentation for the developers.
```
Comment (longgamma-massa) [inline profile/README.md:46]: I doubt that we should mention 2 documentation repos, as the docu-dev should superseed massa-docs, if I am not mistaken. @gregLibert Pls validate if we should mention both repos in Massa organization Readme.

**Review by loonggamma — CHANGES_REQUESTED** [review]

File: profile/README.md (hunk 10 above)
Comment (litchipi) [reply profile/README.md:24]: Yeah, I don't know all the repos so I had to do with what it looked like.
Something like "Repository for smart-contracts standards like NFT and ERC-20, and all Massa standards" ?

File: profile/README.md (hunk 8 above)
Comment (AurelienFT) [reply profile/README.md:19]: I'm personally ok with that

File: profile/README.md [hunk 12]
```diff
@@ -1 +1,45 @@
...
+- [massa-sc-examples](https://github.com/massalabs/massa-sc-examples): Smart-contract examples.
+- [as](https://github.com/massalabs/as): Utils to write smart contracts in AssemblyScript.
+
+## Clients
+- [Thyra](https://github.com/massalabs/thyra): User-friendly client connected to the Massa blockchain to perform common operations. Accepts plugins to add more fonctionalities.
+- [thyra-plugin-store](https://github.com/massalabs/thyra-plugin-store): MassaLabs's list official Thyra plugins supported.
+- [thyra-plugin-wallet](https://github.com/massalabs/thyra-plugin-wallet): Plugin for Thyra implementing a wallet.
+- [thyra-plugin-blogNoCode](https://github.com/massalabs/thyra-plugin-blogNoCode): Plugin for Thyra to create a blog without coding, allowing to store the website on the blockchain.
+- [thyra-node-manager-plugin](https://github.com/massalabs/thyra-node-manager-plugin): Plugin for Thyra to manage a remote node from a user-friendly interface.
+- [thyra-playground-plugin](https://github.com/massalabs/thyra-playground-plugin): Plugin for Thyra to serve a playground for smart-contracts development.
```
Comment (SlnPons) [inline profile/README.md:32]: To delete or archive. Do you agree @longgamma-massa & @gregLibert?

Comment (gregLibert) [reply profile/README.md:32]: you want to delete the repo or the content of the readme ?

**Comment by qdrn** [comment]
Thanks for the PR. However I think this PR misses the point. This README shouldn't be an exhaustive list of all the repos at Massa but rather a short intro to Massa and point the reader to relevant links/repo. Otherwise it will most probably confuse users. Don't forget that this README will be displayed here: https://github.com/massalabs on top of pinned repos.

**Comment by litchipi** [comment]
No problem ! However I'd like to know what is expected to be in this README, as I guess the idea is to be non redundant with the official website of Massa, the massa-docs, massa handbook, etc...
As I had a bit of trouble to navigate through all these repositories when onboarding, I thought it was a good place. It's okay if you think it doesn't have a place here, but I think this information should be somewhere (handbook maybe ?).

I'm closing this for now, let's discuss what should be written in the org's README in the initial issue: https://github.com/massalabs/massa/issues/3467

**Pull request closed without merging**